core/job/handler/v1beta1/job.go

```diff
@@ -524,6 +525,41 @@ func (jh *JobHandler) UpdateJobsState(ctx context.Context, req *pb.UpdateJobsSta
 	return &pb.UpdateJobsStateResponse{}, nil
 }
 
+func (jh *JobHandler) SyncJobsState(ctx context.Context, req *pb.SyncJobsStateRequest) (*pb.SyncJobsStateResponse, error) {
```

Comment:
need to understand the scope here. what is the added value of `SyncJobsState`, compared to `UpdateJobsState`?

in `SyncJobsState`, any reason why we don't fetch the status from the scheduler directly?

Reply:
Indeed, Sync job state will be fired from the scheduler directly